wadas/domain/animal_detection_mode.py

```diff
@@ -46,11 +47,44 @@ def run(self):
                 ):
                     logger.info("Instantiating FTPS server...")
                     self.ftp_thread = FTPsServer.ftps_server.run()
+        logger.info("Ready for video stream from Camera(s)...")
 
-        self.start_actuator_server()
+    def _detect(self, cur_image_path):
+        results, detected_img_path = self.ai_model.process_image(cur_image_path, True)
+        self.last_detection = detected_img_path
+        return results, detected_img_path
+
+    def _format_classified_animals_string(self, classified_animals):
+        # Prepare a list of classified animals to print in UI
+        if self.last_classified_animals_str:
```

Comment:
Isn't this control redundant? I assume that the goal is to initialize / reset the last classification. So, the following line should be enough.